src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -82,6 +82,9 @@ public override bool Execute()
         protected Regex CreateXmlUpdateRegex(string elementName, string contentGroupName) =>
             new Regex($@"<{elementName}>(?<{contentGroupName}>.*)</{elementName}>");
 
+        protected Regex CreateMSBuildSdkUpdateRegex(string msbuildSdkName, string contentGroupName) =>
+            new Regex($@"""{msbuildSdkName}"": ""(?<{contentGroupName}>.*)""");
```

Comment:
For the `": "` match, consider being friendly to bad formatting, like `":"` or `" : "`. A manual bad change might get past review and it'd be nice for auto-update to keep working without a manual fix.

Reply:
Fixed. Now looks for 0+ whitespace characters.